task/09-functions-n-closures-tasks.js

```diff
@@ -132,7 +148,13 @@ function retry(func, attempts) {
  *
  */
 function logger(func, logFunc) {
-    throw new Error('Not implemented');
+    return function () {
+        let arg = Array.from(arguments).map(e => JSON.stringify(e)).join()
```

Comment:
with `rest parameters` you even don't need the `Array.from`. The same for functions bellow